manim/mobject/geometry.py

```diff
@@ -265,60 +277,74 @@ def stop_angle(self):
         return angle_of_vector(self.points[-1] - self.get_arc_center()) % TAU
 
 
+@attr.s(auto_attribs=True, eq=False)
 class ArcBetweenPoints(Arc):
     """
     Inherits from Arc and additionally takes 2 points between which the arc is spanned.
     """
 
-    def __init__(self, start, end, angle=TAU / 4, radius=None, **kwargs):
-        if radius is not None:
-            self.radius = radius
-            if radius < 0:
+    start: tp.Any = None
+    end: tp.Any = None
```

Comment:
iterable, as it just does `np.array(start)` and `np.array(end)`
Although I guess we could simply enforce the usage of `np.ndarray`

```suggestion
    start: tp.Iterable[tp.Union[int, float]] = None
    end: tp.Iterable[tp.Union[int, float]] = None
```
```suggestion
    start: np.ndarray = None
    end: np.ndarray = None
```